tests/test_recv.py

```
@@ -0,0 +1,172 @@
+		import io
+except Exception as err:
+	raise ImportError("[CWE-758] Failed to import test context") from err
+
+
+@context.markWithMetaTag("extra", "coverage")
+class McastRECVTestSuite(context.BasicUsageTestSuite):
+	"""Test cases for McastRECV class doStep method."""
+
+	__module__ = "tests.test_recv"
+
+	def setUp(self):
```

Ruff (0.8.2)

Missing return type annotation for public function `setUp`

Add return type annotation: `None`

(ANN201)